### ISSUE TITLE
What to do with multi-cycle isotherms?

### ISSUE THREAD

**Issue by danieleongari** [NIST-ISODB/isotherm-digitizer-panel#55]
I bumped into this paper that reports a study of NH3 adsorption in MOF NU-300:
https://pubs.acs.org/doi/10.1021/acsanm.9b01534
![image](https://user-images.githubusercontent.com/25899092/94004229-a6008980-fd9c-11ea-827b-286ca5515abd.png)

Just a summary of what is going on:
* NU-300 has "uncoordinated" COO groups from the ligands that potentially chemisorb NH3 in the first cycle
* the crystallinity is retained after the first cycle
* in the second and third cycle the NH3 is physisorbing, with a lower working capacity
* the authors report that after the third cycle the material loses crystallinity

In this case, my choice would be to discard the digitization of this paper because the material is decomposing.
But assuming that the material is not decomposing (and therefore isotherms are more likely reproducible) we have a total of 9 isotherms from the same reference, material, gas and temperature:
* it is fine to report separately the curves from the two different pressure ranges, they would superimpose when compared, and this multiple-resolution/range isotherms are already present in the database
* at the moment the digitizer does not have the option adsorption/desorption to distinguish them
* does it makes sense to have also a new field, e.g., `NOTES` to report that they are cycle 1/2/3? How to deal with cycles? Is this an important information we want to include if present?

I want to raise the attention on this point, because I would be certainly confuse if I see in the database 9 isotherms for the same paper/material/gas/temperature.

@dwsideriusNIST please, let me know your opinion, and what did you do with materials that by admission of the authors decompose upon adsorption. Thanks!

**Comment by dwsideriusNIST** [comment]
@danieleongari, these are all really good points to raise. I'll address them in separate comments

First, a general comment: this paper reveals how an adsorption isotherm is dependent on the history of an adsorbent-adsorbate system; it can only be considered a property of that system under very specific preparation and handling conditions. In the reference isotherms coordinated by NIST (2018 and one that came out last week), there are very specific instructions for material preparation prior to the adsorption/desorption cycling.

**Comment by dwsideriusNIST** [comment]
1) Regarding a `NOTES` field: my student digitizers used the `custom_data` field for comments like this. Re-implementing the `custom_data` field is on the digitizer to-do list

**Comment by dwsideriusNIST** [comment]
2) RE distinguishing between adsorption and desorption branches: the absence of this feature reflects a choice made early in the development spec of ISODB. It's been pointed out as a deficiency by @pauliacomi in his work utilizing on ISODB; he did write some code to attempt to automatically detect adsorption and desorption branches.

I would suggest that _for now_ we do not distinguish between branches. Perhaps a future spec of the digitizer could have parallel fields so that adsorption and desorption can be explicitly identified. Post-processing could recombine them for ISODB, or maybe by then I'll have enough resources to properly handle the different branches in ISODB.

**Comment by dwsideriusNIST** [comment]
3) Whether the data should be included or not: I have allowed data of this type (e.g., different results for different cycles, material degradation, etc.) in the past and will continue to do so. There is a certain amount of _caveat lector_ involved in using the data from ISODB and the user is strongly encouraged to consult the original literature for cases where the isotherms are non-intuitive or apparently contradictory.

**Comment by danieleongari** [comment]
Thank you very much @dwsideriusNIST for your insightful comments: I will take them into account for the digitation. For sure in our "refined" subset of isotherm aiming to be compared with simulated data, we may want to think carefully about these points!

**Comment by ltalirz** [comment]
By the way, in case you are wondering: the repository is private, i.e. I believe @ mentioning people who aren't contributors will not result in a notification.

Since this repository contains scripts to set up the old digitizer, perhaps the open issues could be moved to the new digitizer repo?

**Comment by dwsideriusNIST** [comment]
Yes, let's move everything over to @ltalirz/isotherm-digitizer-panel!

**Comment by ltalirz** [comment]
If we want to use github's process for transferring issues, [it can only be done within the same user/organisation and only between public repositories](https://docs.github.com/en/github/managing-your-work-on-github/transferring-an-issue-to-another-repository).

I was anyhow wondering - where do you intend the isodb library to go?
Shall we quickly set up a "nist-isodb" organisation, where we collect all of these repos?

In terms of making things public, I'm happy to make the panel app public, as well as the ansible scripts for setting it up (and move them to a common github org).
